metricbeat/module/haproxy/info/info.go

```diff
@@ -19,40 +19,30 @@ var (
 	debugf = logp.MakeDebug("haproxy-info")
 )
 
-// init registers the MetricSet with the central registry.
-// The New method will be called after the setup of the module and before starting to fetch data
+// init haproxy info metricset
 func init() {
 	if err := mb.Registry.AddMetricSet("haproxy", "info", New); err != nil {
 		panic(err)
 	}
 }
 
-// MetricSet type defines all fields of the MetricSet
-// As a minimum it must inherit the mb.BaseMetricSet fields, but can be extended with
-// additional entries. These variables can be used to persist data or configuration between
-// multiple fetch calls.
+// MetricSet for haproxy info
```

Comment:
Please 🙏  end the godocs with periods. Then no one has to wonder if you forgot to write the end of the sentence. 😼